test/test_integration_credentials_netrc.py

```diff
@@ -393,7 +365,7 @@ def test_https_can_load_policy_netrc(self,mock_accessible):
             utils.assert_set_and_get(self, variable)
 
     @integration_test()
-    def test_secret_netrc(self,mock_accessible):
```

Comment:
GENERAL NOTE: please go through the test files and remove extra lines and fix alignment 🙏